paddle/fluid/framework/executor.cc

```diff
@@ -203,29 +206,38 @@ static bool has_feed_operators(
     if (op->Type() == kFeedOpType) {
       feed_count++;
       // The input variable's name of feed_op should be feed_holder_name.
-      PADDLE_ENFORCE_EQ(op->Input("X")[0], feed_holder_name,
-                        "Input to feed op should be '%s'", feed_holder_name);
+      PADDLE_ENFORCE_EQ(
+          op->Input("X")[0], feed_holder_name,
+          platform::errors::PreconditionNotMet(
+              "Input to feed op should be '%s'", feed_holder_name));
```

Comment:
May be better to add message "but received XX".

Reply:
Done